engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -3556,7 +3556,8 @@ public void reallyRun() {
 
                 final List<Long> networkIds = _networksDao.findNetworksToGarbageCollect();
                 final int netGcWait = NumbersUtil.parseInt(_configDao.getValue(NetworkGcWait.key()), 60);
-                logger.info("NetworkGarbageCollector uses '{}' seconds for GC interval.", netGcWait);
+                final int netGcInterval = NumbersUtil.parseInt(_configDao.getValue(NetworkGcInterval.key()), 60);
```

Comment:
Is the purpose to only log it, why not use the ConfigKey read the value, than use the ConfigDao?

Reply:
```suggestion
                final int netGcInterval = NumbersUtil.parseInt(NetworkGcInterval.value()), 60);
```

Reply:
@andrijapanicsb ?

Reply:
or even more simple:
```
final int netGcWait = NetworkGcWait.value();
final int netGcInterval = NetworkGcInterval.value();
```